### ISSUE TITLE
[v6] Auto complete not working with vscode

### ISSUE THREAD

**Issue by r0b0t3d** [invertase/react-native-firebase#2648]
## Issue

Hi, I got issue with autocomplete in v6.

`package.json`
```
{
    "dependencies": {
        "@react-native-firebase/analytics": "^6.0.0",
        "@react-native-firebase/app": "^6.0.0",
        "@react-native-firebase/auth": "^6.0.0",
        "@react-native-firebase/crashlytics": "^6.0.0",
        "@react-native-firebase/database": "^6.0.0",
        "@react-native-firebase/storage": "^6.0.0",
    }
}
```

```
import database from '@react-native-firebase/database';

const usersRef = database().ref('users');
```
<img width="338" alt="image" src="https://user-images.githubusercontent.com/8109285/65776327-18c82c00-e16c-11e9-9b9a-ef34ff5eada3.png">
As you can see in the above image, vs code can not determine the type of `usersRef`. It happens for other packages as well.

**Comment by Ehesp** [comment]
Are you using typescript?

**Comment by r0b0t3d** [comment]
I am using `"typescript": "^3.6.3"`

**Comment by Ehesp** [comment]
Interesting, I also see this. While we investigate try the following which seems to be working fine:

```
import firebase from '@react-native-firebase/app';

...
const usersRef = firebase.database().ref('users');
```

**Comment by r0b0t3d** [comment]
@Ehesp it is working with your suggestion.
But we still need to make it works with `@react-native-firebase/database`, do we?

**Comment by Ehesp** [comment]
Yeah we do, trying to figure out both why it's not working and how to get it working.

**Comment by mtford90** [comment]
Experiencing the same with Webstorm - for both auth & database.

**Comment by brunolemos** [comment]
### Workaround

Import both `@react-native-firebase/app` and the module package:

```diff
+import '@react-native-firebase/analytics'
import firebase from '@react-native-firebase/app'

firebase.analytics().setUserId('123')
```

#### Before

![image](https://user-images.githubusercontent.com/619186/66262490-a08ae780-e7b7-11e9-8430-7b908258fd6a.png)

#### After

![image](https://user-images.githubusercontent.com/619186/66262493-b00a3080-e7b7-11e9-88b7-2654da94e66b.png)

**Comment by Salakar** [comment]
Hey 👋 this issue should now be fixed in v6.0.1, please try it out - thanks. [release notes](https://invertase.io/oss/react-native-firebase/releases/v6.0.1)

**Comment by swizes** [comment]
the problem still occurs in webstorm
